static/js/global_init.js

```diff
@@ -79,7 +79,11 @@ process.on('unhandledRejection', reason => { // eslint-disable-line no-unused-va
 
 // fix 'unknown prop' error
 import injectTapEventPlugin from 'react-tap-event-plugin';
-injectTapEventPlugin();
+
+try {
```

Comment:
Why was this needed?

Reply:
it throws an error if it's already been called

Reply:
(and this file is basically re-required every time mocha goes to run the tests)

Reply:
Can you add a comment to explain the try block?